pkg/internal/mock/wallet/mock_wallet.go

```diff
@@ -45,3 +48,8 @@ func (m *CloseableWallet) PackMessage(envelope *wallet.Envelope) ([]byte, error)
 func (m *CloseableWallet) UnpackMessage(encMessage []byte) (*wallet.Envelope, error) {
 	return nil, nil
 }
+
+// NewDID returns new DID Document
+func (m *CloseableWallet) NewDID() (*did.Doc, error) {
```

Comment:
We should be able to specify the DID method.

Reply:
Will be adding it in upcoming PR under same issue number, added a TODO with issue ID for now.